dallinger/command_line.py

```diff
@@ -218,7 +218,6 @@ def verify_package(verbose=True):
         os.path.join("templates", "launch.html"),
         os.path.join("templates", "thanks.html"),
         os.path.join("static", "css", "dallinger.css"),
-        os.path.join("static", "scripts", "dallinger.js"),
```

Comment:
Not sure about this one. This is to check that the custom experiment does not include a file called "dallinger.js", which might still be a good idea, especially if we later want to rename "dallinger2.js" to "dallinger.js".

Reply:
OK with moving forward.